### ISSUE TITLE
Add toggle for log vs. linear axes

### ISSUE THREAD

**Issue by brittsikora** [RhoInc/safety-shift-plot#3]
Auto-scaling was throwing people off, so add ability to set limits. Or toggle between log axis and linear.

**Comment by jwildfire** [comment]
Might be handled by #9.

**Comment by samussiah** [comment]
I like @brittsikora 's second solution.  Perhaps we could add a control to flip the axis scale.  #9 involves maintaining a consistent axis whilst filtering.  #15 will take care of that once #18 is merged in.

**Comment by jwildfire** [comment]
I'd say adding a log/linear toggle is nice to have, but I'm thinking it might be a little bit complex for this one ... or maybe not ... At any rate, I'm updating the title here to reflect that goal and bumping this to 1.3.
